dds/DCPS/RTPS/SecurityHelpers.h

```diff
@@ -9,16 +9,20 @@
 #define OPENDDS_SECURITY_HELPERS_H
 
 #include "dds/DdsSecurityCoreC.h"
+#include "dds/DdsDcpsGuidC.h"
```

Comment:
Was this required for something?

Reply:
I was doing something with this header in a more isolated context and it didn't have `EntityId_t`. I ended up scrapping what I was doing, but kept this because it's correct and someone might have the same problem in the future.

Reply:
If by "correct" you mean "an unnecessary burden on compilation," then I agree. :)

Reply:
I mean we ARE using C++, the king of burdens when it comes to compilation